src/card-validator/card-validator.spec.ts

```diff
@@ -2970,4 +2970,26 @@ describe('card-validator', () => {
       expect(result.card?.niceType).toBe(card.issuer)
     })
   })
+
+  it('should be a correctly mask for a card number', () => {
+    cards.forEach((card) => {
+      const result = valid.number(card.cardNumber)
+
+      if (!result.card?.type) return
+
+      const mask = maskCardNumber(result.card?.type)
```

Comment:
```suggestion
      const mask = maskCardNumber(result.card?.type ?? '')
```

Se você aceitar a sugestão do assert você precisa fazer essa alteração aqui

Reply:
alterei o tipo da propriedade pra receber undefined e fiz o tratamento de lançar o erro nela